test/e2e-common.sh

```diff
@@ -42,8 +46,8 @@ function test_setup() {
 
   # Wait for pods to be running.
   echo ">> Waiting for Kourier components to be running..."
-  wait_until_pods_running kourier-system || return 1
-  wait_until_service_has_external_http_address kourier-system kourier || return 1
+  wait_until_pods_running "${KOURIER_NAMESPACE}" || return 1
+  wait_until_service_has_external_ip "${KOURIER_NAMESPACE}" kourier || return 1
```

Comment:
This reverts a change that was introduced a few weeks ago: https://github.com/knative/net-kourier/pull/58/commits/205bd4033e7ef70fb0abc64b5641f97536073b97
Should we use `wait_until_service_has_external_http_address` instead?

Reply:
Yeah. Sorry about that. It was an oversight. Here's the PR: https://github.com/knative/net-kourier/pull/81